db.js

```diff
@@ -0,0 +1,51 @@
+const knex = require('knex');
+const knexConfig = require('./knexfile.js');
+const db = knex(knexConfig.development);
+
+module.exports = {
+    getDishes,
+    addDish,
+    getDish,
+    getRecipes,
+    addRecipe,
+    getRecipe,
+    getShoppingList
+};
+
+function getDishes(){
+    return db('dishes')
+}
+
+function getDish(id){
+    return db('dishes').where({ id: Number(id) });
+}
+
+function addDish(dish) {
+    return db('dishes')
+    .insert(dish)
+    .then(ids => ({ id: ids[0] }));
```

Comment:
You could alternatively use array destructuring here and in the other callbacks that are identical:
```javascript
.then(([ id ]) => ({ id }))
```